.github/workflows/build.yml

```diff
@@ -1,35 +1,28 @@
-name: 'Build Thrift Compiler and Libraries'
+name: 'Build'
+
 on:
   push:
     branches: [ '*' ]
   pull_request:
     branches: [ '*' ]
+
+env:
+  BUILD_DEPS: automake bison flex g++ git libboost-all-dev libevent-dev libssl-dev libtool make pkg-config
```

Comment:
Clever! I didn't know about `env`

Reply:
Thanks! Do you mind merging this some time later because I don't have write access